doc/build-helpers/fetchers.chapter.md

```diff
@@ -70,28 +126,32 @@ fetchurl {
 }
 ```
 
-When building the recipe, you'll get an error message that you can use to determine the correct hash:
+When building the package, use the error message to determine the correct hash:
 
 ```shell
 $ nix-build
-(some output removed for clarity)
+...
```

Comment:
I prefer to be verbose in this case: most people who are familiar with CLIs will understand both "..." and "(some output removed for clarity)", but there is a small amount of less experienced folks who will get a wall of text of output and think something went wrong and never actually read the output.

Using "(some output removed for clarity)" gives them indirect encouragement to read through the wall of text until they find the text they should be reading.

```suggestion
(some output removed for clarity)
```

Reply:
Hm, I'd rather stick to the widely used convention and (at some point in the future) add a page on how to read examples.

Reply:
I decided to defer on this one since there is already a bunch of content (from previous PRs) where I used `(some output removed for clarity)` or `(output removed for clarity)`, there are also places in the manual where `...` is used, and other places where nothing is used.

I'd like to make this part of the linting tool for docs content, which is in [the roadmap](https://discourse.nixos.org/t/a-roadmap-for-the-documentation-ecosystem/42328) for docs, so let's make this decision in the issue for the linting tool itself. And then we can apply it to all existing content to make things consistent at once.